components/next_steps_view/steps.ts

```diff
@@ -45,60 +48,60 @@ export type StepType = {
 export const Steps: StepType[] = [
     {
         id: RecommendedNextSteps.COMPLETE_PROFILE,
-        title: localizeMessage(
-            'next_steps_view.titles.completeProfile',
-            'Complete your profile',
-        ),
+        title: {
+            titleId: t('next_steps_view.titles.completeProfile'),
```

Comment:
@stevemudie noticed these weren't being translated. When I looked into it, my best guess is that since these steps are define before loading data into redux and localizeMessage depends on internationalization data being present in redux, we were just getting the default message all the time. So we can fix it by deferring translating to render time instead of initial load time.

Reply:
Good catch!